# Neural_Network_Charity_Analysis

After updating tensorflow, I recieved this error message when trying to run the first cell in jupyter notebook. After many attempts at fixing the issue locally, I decided to use google colab to complete the challenge. 

<img width="725" alt="Screen Shot 2021-11-16 at 9 23 19 PM" src="https://user-images.githubusercontent.com/84936545/142100367-41b64bdd-0d12-4681-b7d6-9743f2ef5241.png">
